### PR TITLE
add author's preprint for Steinmach et al

### PR DESCRIPTION
**Pull request by pdebuyl**
Otherwise, the final pdf is also available from another author's homepage but without direct link to bibliographical info http://uses.icomp.ufam.edu.br/index.php/publicacoes

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -409,7 +405,7 @@ but discussion of those is outside the scope of this paper.
 [noble-rules]: http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000424
 [repo-url]: https://github.com/swcarpentry/good-enough-practices-in-scientific-computing
 [sandve-reproducible]: http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003285
-[steinmacher-newcomers]: FIXME
+[steinmacher-newcomers]: http://lapessc.ime.usp.br/work.jsf?p1=15673 "Steinmacher et al, The hard life of open source software project newcomers, Procs. 7th Int. Workshop on Cooperative and Human Aspects of Software Engineering. ACM, 2014."
```
Comment (gvwilson) [inline index.md:408]: I think everything after the colon is used as the URL in the link, so the full citation won't work here. Can you just give us the link, and we'll figure out the bibliography when we convert to a proper paper? (I'm still annoyed that Markdown doesn't have a standard format for citations...)

**Comment by pdebuyl** [comment]
The syntax is from http://daringfireball.net/projects/markdown/syntax#link

Pandoc renders it correctly and github also does, see the stored version on my branch at https://github.com/pdebuyl/good-enough-practices-in-scientific-computing/blob/steinmach-link/index.md

This is standard compliant and allows to store the citation. The text in quotes appears as the alt-text of the URL. Anyway, I'll gladly abide to simplicity if the above details are not motivating enough.

**Comment by pdebuyl** [comment]
Also, pandoc-citeproc https://github.com/jgm/pandoc-citeproc allows to use a bibfile with pandoc. But this implies choosing pandoc for working, which may not be desired.

**Comment by gvwilson** [comment]
I did not know that worked - and it works with Jekyll too (which is what 
GitHub uses for rendering).  I'm learning every day :-)
thanks,
Greg

**Comment by gvwilson** [comment]
I'd like to stick to Jekyll (which uses Kramdown by default for 
parsing), since that's what GitHub uses.

**Pull request merged**